storage/tests/StorageManager_unittest.cpp

```diff
@@ -205,46 +205,4 @@ TEST(StorageManagerTest, DifferentNUMANodeBlobTestWithEviction) {
 }
 #endif  // QUICKSTEP_HAVE_LIBNUMA
 
-// Trigger an eviction from the same shard in StorageManager's
```

Comment:
@zuyu It was because the scenario for which it was designed to test was eliminated by the changes to StorageManager. It was testing a thread interleaving situation where a thread would try to acquire a lock which it already owned. 

When refactoring `makeRoomForBlock` there wouldn't be an attempt to acquire that a second lock. We removed the argument in `makeRoomForBlock(... , const block_id locked_block_id)` because the caller would never have a locked block going in. The unit test was designed to capture that exact scenario.